src/core_ocean/Registry.xml

```diff
@@ -1657,6 +1637,7 @@
 			<var name="xtime"/>
 			<var name="simulationStartTime"/>
 			<var name="boundaryLayerDepth"/>
+			<var name="indexBoundaryLayerDepth"/>
```

Comment:
Same thing here. My restart test passes using QU240 and cvmix on, using kpp, with both `boundaryLayerDepth` and `indexBoundaryLayerDepth` removed. Are there conditions where those variables are used from the previous timestep, rather than recomputed?

Reply:
agreed.  I think this was from your first tapering implementation when it was based on BLD.  I'm sure it can be removed.

Reply:
What about `boundaryLayerDepth` in the restart file? That was added a long time ago. Probably before the cvmix:
```
cb8195861b (Douglas Jacobsen   2015-02-02 13:45:52 -0700 1659)          <var name="boundaryLayerDepth"/>
```

Reply:
I can't think of any reason to include boundaryLayerDepth in the restart file.  It is always computed before mixing and is only used within the mixing routine.  So I don't think that should affect restarts.